README.md

```diff
@@ -6,7 +6,9 @@
 
 > One paragraph statement about the project.
 
```

Comment:
- [ ] [OPTIONAL] Please, try to write down a more informative explanation about your project.